jenkins/java/ql/src/FileFromRestOfPath.md

```diff
@@ -1 +1,28 @@
-`#getRestOfPath` and/or the file path resulting from its use needs to be carefully sanitized to not result in a path traversal vulnerability.
+The value from `#getRestOfPath` is used to build a `File`, resulting in a potential path traversal vulnerability.
+
+## Why is this a problem?
+
+If your code is modifying or reading a file as specified in the URL, users able to access this URL may be able to access files outside your intended root directory unless you're restricting allowed values.
```

Comment:
Phrase is a trifle too long. But can't propose anything better. Thought of putting "able to access this URL" between commas. But this doesn't work well with the first segment.

So this is good to go.